foxpuppet/windows/__init__.py

```diff
@@ -0,0 +1,54 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+
+
+class WindowManager(object):
```

Comment:
I know this is from my code, but I was being lazy by putting things into `__init__.py`. Could you create separate modules for `WindowManager` and `BaseWindow`.